### PR TITLE
Fix for translations

### PR DESCRIPTION
**Pull request by garubi**
1. Enclosed the "Agree to terms" string in _e() function
2. Changed some _e() to honor the (maybe) changed edd_label (use edd_get_label_singular() )

### CODE REVIEW

**Comment by garubi** [comment]
Any news on this?

File: edd-terms-per-product.php [hunk 1]
```diff
@@ -55,11 +55,12 @@ public function metabox_field( $post_id = 0 ) {
 		$terms = get_post_meta( $post_id, '_edd_download_terms', true );
 		?>
 		<p>
-			<strong><?php _e( 'Download Terms of Use', 'edd-terms-per-product' ); ?></strong>
+			<strong><?php printf( __( '%s Terms of Use:', 'edd-terms-per-product' ), edd_get_label_singular() ); ?></strong>
```
Comment (robincornett) [inline edd-terms-per-product.php:58]:
```suggestion
			<strong><?php printf( esc_html__( '%s Terms of Use:', 'edd-terms-per-product' ), esc_html( edd_get_label_singular() ) ); ?></strong>
```

File: edd-terms-per-product.php [hunk 2]
```diff
@@ -55,11 +55,12 @@ public function metabox_field( $post_id = 0 ) {
...
 		</p>
 		<p>
 			<textarea name="_edd_download_terms" id="edd_download_terms" rows="10" cols="50" class="large-text"><?php echo $terms; ?></textarea>
-			<label for="edd_download_terms"><?php _e( 'Enter the terms of use for this product.', 'edd-terms-per-product' ); ?></label>
+			<label for="edd_download_terms"><?php printf( __( 'Enter the terms of use for this %s:', 'edd-terms-per-product' ), edd_get_label_singular() ); ?></label>
```
Comment (robincornett) [inline edd-terms-per-product.php:62]:
```suggestion
			<label for="edd_download_terms"><?php printf( esc_html__( 'Enter the terms of use for this %s:', 'edd-terms-per-product' ), edd_get_label_singular() ); ?></label>
```

File: edd-terms-per-product.php [hunk 3]
```diff
@@ -100,11 +101,11 @@ public function product_terms() {
 						<?php echo wpautop( $terms ); ?>
 					</div>
 					<div id="edd_show_<?php echo $item['id']; ?>_terms">
-						<a href="#edd_<?php echo $item['id']; ?>_terms" class="edd_per_product_terms_links"><?php printf( __( 'Show Terms For %s', 'edd' ), get_post_field( 'post_title', $item['id'] ) ); ?></a>
-						<a href="#edd_<?php echo $item['id']; ?>_terms" class="edd_per_product_terms_links" style="display:none;"><?php _e( 'Hide Terms', 'edd' ); ?></a>
+						<a href="#edd_<?php echo $item['id']; ?>_terms" class="edd_per_product_terms_links"><?php printf( __( 'Show Terms For %s', 'edd-terms-per-product' ), get_post_field( 'post_title', $item['id'] ) ); ?></a>
```
Comment (robincornett) [inline edd-terms-per-product.php:104]:
```suggestion
						<a href="#edd_<?php echo $item['id']; ?>_terms" class="edd_per_product_terms_links"><?php printf( esc_html__( 'Show Terms For %s', 'edd-terms-per-product' ), esc_html( get_post_field( 'post_title', $item['id'] ) ) ); ?></a>
```

File: edd-terms-per-product.php [hunk 4]
```diff
@@ -100,11 +101,11 @@ public function product_terms() {
...
+						<a href="#edd_<?php echo $item['id']; ?>_terms" class="edd_per_product_terms_links" style="display:none;"><?php _e( 'Hide Terms', 'edd-terms-per-product' ); ?></a>
```
Comment (robincornett) [inline edd-terms-per-product.php:105]:
```suggestion
						<a href="#edd_<?php echo $item['id']; ?>_terms" class="edd_per_product_terms_links" style="display:none;"><?php esc_html_e( 'Hide Terms', 'edd-terms-per-product' ); ?></a>
```

File: edd-terms-per-product.php [hunk 5]
```diff
@@ -100,11 +101,11 @@ public function product_terms() {
...
 					</div>
 					<input name="edd_agree_to_terms_<?php echo $item['id']; ?>" class="required" type="checkbox" id="edd_agree_to_terms_<?php echo $item['id']; ?>" value="1"/>
-					<label for="edd_agree_to_terms_<?php echo $item['id']; ?>">Agree to Terms</label>
+					<label for="edd_agree_to_terms_<?php echo $item['id']; ?>"><?php _e( 'Agree to Terms', 'edd-terms-per-product' ); ?></label>
```
Comment (robincornett) [inline edd-terms-per-product.php:108]:
```suggestion
					<label for="edd_agree_to_terms_<?php echo $item['id']; ?>"><?php esc_html_e( 'Agree to Terms', 'edd-terms-per-product' ); ?></label>
```

**Review by robincornett — CHANGES_REQUESTED** [review]

**Review by ashleyfae — APPROVED** [review]

**Review by robincornett — APPROVED** [review]

**Pull request merged**